frontend/dogplus-frontend/src/components/Post.tsx

```diff
@@ -10,6 +11,7 @@ interface PostProps {
 export const PostCard: React.FC<PostProps> = ({ post, isLiked }) => {
     const [liked, setLiked] = React.useState(isLiked);
     const [likeCount, setLikeCount] = React.useState(post.like_count);
+    const navigate = useNavigate();
```

Comment:
Should we add some sort of pagination so that we do not fetch all posts right away, but get the first then etc?

Reply:
Maybe create another issue for this?